airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/resources/spec.json

```diff
@@ -2,7 +2,7 @@
   "documentationUrl": "https://docs.airbyte.com/integrations/destinations/bigquery",
   "supportsIncremental": true,
   "supportsNormalization": false,
-  "supportsDBT": true,
+  "supportsDBT": false,
```

Comment:
same as my first comment - this change should not be part of this PR, so please rebase on master